<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentView</class>
 <widget class="QWidget" name="AppointmentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预约管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="tstSearch">
       <property name="placeholderText">
        <string>请输入患者ID或医生ID进行查找</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbStatusFilter"/>
     </item>
     <item>
      <widget class="QPushButton" name="btSearch">
       <property name="text">
        <string>查找</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/search.png</normaloff>:/img/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAdd">
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btDelete">
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btEdit">
       <property name="text">
        <string>修改</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/edit.png</normaloff>:/img/edit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btApprove">
       <property name="text">
        <string>通过审核</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background-color: #4CAF50; color: white; }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btReject">
       <property name="text">
        <string>拒绝预约</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background-color: #f44336; color: white; }</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lab4.qrc"/>
 </resources>
 <connections/>
</ui>
